src/test/java/ttt/BoardSizeTest.java

```diff
@@ -0,0 +1,37 @@
+package ttt;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class BoardSizeTest {
+
+    @Parameterized.Parameters
+    public static Collection<Object> data() {
+        return Arrays.asList(new Object[][]{
+                { 3, 9 }, { 4, 16 }
+        });
+    }
+
+    private int fInput;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2639507600/issues/source?bid=3284885&fileBranchId=3321123#l24)